modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -19,10 +19,11 @@ In the table, features are marked with the following statuses:
 In {OCP} 4.13, the `stable` channel has been removed. Before upgrading to {OCP} 4.13, if you are already on the `stable` channel, choose the appropriate channel and switch to it.
 ====
 
+[options="header"]
 |===
-|*OpenShift GitOps* 7+|*Component Versions*|*OpenShift Versions*
+|OpenShift GitOps 7+|Component Versions|OpenShift Versions
 
-|*Version* |*`kam`*    |*Helm*  |*Kustomize* |*Argo CD*|*Argo Rollouts*|*Dex*     |*RH SSO* |
+|Version |`kam`    |Helm  |Kustomize |Argo CD |Argo Rollouts |Dex     |RH SSO |
```

Comment:
@Srivaralakshmi not related to this PR, should we also add argocd CLI version in this table?

cc: @anajosep

Reply:
sure, let us discuss it with ANand and take a call. Thanks, @varshab1210